css/index.css

```diff
@@ -66,7 +67,22 @@ p {
     line-height: 1.4;
 }
 
+img {
+    image-rendering: auto;
+    image-rendering: crisp-edges;
+    /* image-rendering: pixelated; */
+}
+
+header {
```

Comment:
You have some duplicate CSS in here, the last one will take precedence.  You can also work on your header here to get it matched up width wise to the rest of the page.